components/utils.py

```diff
@@ -61,4 +61,43 @@ def seconds_to_time(seconds):
     secs = str(int(seconds % 60))
     if len(secs) == 1:
         secs = "0" + secs
-    return mins + ":" + secs
\ No newline at end of file
+    return mins + ":" + secs
+    
+# Send a message using channel object
+async def send_message(channel, msg: str, embedded_msg=None):
+    try:
+        if embedded_msg:
+            await channel.send(msg, embed=embedded_msg)
+        else:
+            await channel.send(msg)
+
+    except Exception as e:
+        print(e)
```

Comment:
Replace print with channel.send. Print outputs to the terminal which is not as visible